openshift/template.yaml

```diff
@@ -5,304 +5,216 @@ labels:
-    namespace: ${THOTH_FRONTEND_NAMESPACE}
+    name: thoth
   stringData:
     app-secret-key: some-secret-key
-
-- apiVersion: v1
-  kind: Secret
-  metadata:
-    name: secret
-    namespace: ${THOTH_FRONTEND_NAMESPACE}
-  stringData:
     thoth-secret: ${THOTH_SECRET}
-
-- apiVersion: v1
-  kind: Secret
-  metadata:
-    name: ceph-secrets
-    namespace: ${THOTH_FRONTEND_NAMESPACE}
-  stringData: &ceph_secrets
     ceph-key-id: ${THOTH_CEPH_KEY_ID}
     ceph-secret-key: ${THOTH_CEPH_SECRET_KEY}
 
-- apiVersion: v1
-  kind: Secret
-  metadata:
-    name: ceph-secrets
-    namespace: ${THOTH_MIDDLEEND_NAMESPACE}
-  stringData:
-    <<: *ceph_secrets
-
-- apiVersion: v1
-  kind: Secret
-  metadata:
-    name: ceph-secrets
-    namespace: ${THOTH_BACKEND_NAMESPACE}
-  stringData:
-    <<: *ceph_secrets
-
-#
-# Network restriction for analyzer containers.
-#
-# TODO: as this does not work on `oc cluter up`, this needs to be configured on actual deployment:
-#   https://blog.openshift.com/whats-new-in-openshift-3-5-network-policy-tech-preview/
-#   https://bugzilla.redhat.com/show_bug.cgi?id=1444144
-#- apiVersion: extensions/v1beta1
-#  kind: NetworkPolicy
-#  metadata:
-#    name: restricted-network-for-analyzer
-#  spec:
-#   podSelector:
-#     matchLabels:
-#       role: analyzer-container
-#   # TODO: whitelist supported registries to pull images from
-#   ingress:
-#   - {}
-#   egress:
-#   - {}
-
-#
-# Namespace quotas
-#
-- apiVersion: v1
-  kind: ResourceQuota
+## Config
+- kind: ConfigMap
+  apiVersion: v1
   metadata:
-    name: mem-cpu-resource-quota
-    namespace: ${THOTH_MIDDLEEND_NAMESPACE}
-  spec:
-    hard:
-      requests.cpu: ${THOTH_MIDDLEEND_CPU_REQUEST}
-      requests.memory: ${THOTH_MIDDLEEND_MEMORY_REQUEST}
-      limits.cpu: ${THOTH_MIDDLEEND_CPU_LIMIT}
-      limits.memory: ${THOTH_MIDDLEEND_MEMORY_LIMIT}
-
-- apiVersion: v1
-  kind: ResourceQuota
+    name: thoth
+  data:
+    middletier_namespace: ${THOTH_MIDDLEEND_NAMESPACE}
+    backend_namespace: ${THOTH_BACKEND_NAMESPACE}
+    ceph-bucket-name: ${THOTH_CEPH_BUCKET}
+    ceph-host: ${THOTH_CEPH_HOST}
+    user-api-service-port: '8080'
+    buildlogs-path: /buildlogs
+    ceph-results-path: /results
+    analyzer-cleanup-time: ${THOTH_ANALYZER_CLEANUP_TIME}
+    sync-observations: ${THOTH_SYNC_OBSERVATIONS}
+    debug-cleanup: ${THOTH_CLEANUP_DEBUG}
+
+## Service Accounts
+- kind: ServiceAccount
+  apiVersion: v1
   metadata:
-    name: mem-cpu-resource-quota
-    namespace: ${THOTH_BACKEND_NAMESPACE}
-  spec:
-    hard:
-      requests.cpu: ${THOTH_BACKEND_CPU_REQUEST}
-      requests.memory: ${THOTH_BACKEND_MEMORY_REQUEST}
-      limits.cpu: ${THOTH_BACKEND_CPU_LIMIT}
-      limits.memory: ${THOTH_BACKEND_MEMORY_LIMIT}
+    name: analyzer
+  automountServiceAccountToken: true
+  
 #
 # Services
-#
 - apiVersion: v1
   kind: Service
   metadata:
     labels:
-      service: thoth-user-api
-    name: thoth-user-api
-    namespace: ${THOTH_FRONTEND_NAMESPACE}
```

Comment:
Did you try to deploy thoth to multiple namespaces? I think this will not work. Here and other parts below.

Reply:
Might by, my focus is to get the Jenkins Pipeline up and running, as that is one namespace anyways... I didnt test it. I have created https://trello.com/c/6V0JAy6L